README.md

```diff
@@ -11,8 +11,9 @@ This builds on top of [ClassicalOrthogonalPolynomials.jl](https://github.com/Jul
 
 For example, the following gives a half-range Chebyshev polynomial:
 ```julia
+julia> usnig SemiclassicalOrthogonalPolynomials
```

Comment:
using

Reply:
That's what I get for thinking I can skip copy-pasting